packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json

```diff
@@ -22,12 +22,26 @@
   "pci_additional_ip_create_step_attach_instance_info_failover_ip": "Cette IP sera directement connectée à l'instance (ancien mode, sans Public Gateway). Veuillez sélectionner une instance à utiliser. Une opération manuelle sur l'instance est nécessaire, <a href=\"{{ guideLink }}\" target=\"_blank\">plus de détails ici</a>",
-  "pci_additional_ip_create_no_instance_message_failover_ip": "Aucune instance pour attacher l'IP de basculement n'a été trouvée. Veuillez en créer un avant.",
+  "pci_additional_ip_create_no_instance_message_failover_ip": "Aucune instance avec réseau privé n'a été détectée dans la région sélectionnée. Merci d'en créer une avant d'attacher une IP flottante.",
 
   "pci_additional_ip_floating_ip": "IP flottantes (Public Cloud)",
   "pci_additional_ip_floating_ip_description": "solution optimisée pour les cas d’utilisation Public Cloud, l’automatisation (Terraform) et le support de réseaux privés régionalisés. Disponible en modèle de facturation à l’usage.",
   "pci_additional_ip_create_step_select_region_floating_ip": "Sélectionnez une région pour votre nouvelle IP flottante",
   "pci_additional_ip_create_step_select_region_description_floating_ip": "Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des produits par région, <a href=\"{{ guideLink }}\" target=\"_blank\">cliquez ici.</a>",
   "pci_additional_ip_create_step_attach_instance_description_floating_ip": "Seules les instances avec des réseaux privés sont affichées",
-  "pci_additional_ip_create_no_instance_message_floating_ip": "Aucune instance avec réseau privé n'a été trouvée dans la région sélectionnée. Veuillez en créer un avant d'attacher une IP flottante."
+  "pci_additional_ip_create_no_instance_message_floating_ip": "Aucune instance avec réseau privé n'a été trouvée dans la région sélectionnée. Veuillez en créer un avant d'attacher une IP flottante.",
+
+  "pci_additional_ip_create_step_summary_banner1": "Aucune Public Gateway utilisable n'a été trouvée dans la région sélectionnée. Une nouvelle sera créée comme suit :",
+  "pci_additional_ip_create_step_summary_banner1_size": "Nouvelle Public Gateway à créer ({{ size }})",
+  "pci_additional_ip_create_step_summary_banner1_price": "Le coût supplémentaire de la Public Gateway s’ajoutera à votre facture : {{ price }} (facturés à l'heure)",
+
+  "pci_additional_ip_create_step_summary_banner2": "Nous avons détecté une Public Gateway déjà utilisée dans la région sélectionnée : {{ gatewayName }}. Une nouvelle IP flottante y sera configurée.",
+
+  "pci_additional_ip_create_step_summary_banner3": "Nous avons détecté une Public Gateway disponible dans la région sélectionnée : {{gatewayName}} qui peut être utilisée. Pour la configurer avec la nouvelle IP flottante, une option de 'Gateway externe' (SNAT) doit être activée sur la Public Gateway. Voulez-vous  continuer ?",
```

Comment:
I think there are two spaces between "Voulez-vous" and "continuer" we can remove one.